crates/wasmtime/src/runtime/store.rs

```diff
@@ -1294,6 +1336,34 @@ impl<T> Store<T> {
     pub fn clear_debug_handler(&mut self) {
         self.inner.debug_handler = None;
     }
+
+    /// Configure a [`Store`] to enable execution recording
+    ///
+    /// This must be initialized before instantiating any module within
+    /// the Store. Recording of events is performed according to provided settings, and
+    /// written to the provided writer.
+    #[cfg(feature = "rr")]
+    pub fn init_recording(
+        &mut self,
+        recorder: impl RecordWriter,
+        settings: RecordSettings,
+    ) -> Result<()> {
+        self.inner.init_recording(recorder, settings)
+    }
+
+    /// Configure a [`Store`] to enable execution replaying
+    ///
+    /// This must be initialized before instantiating any module within
+    /// the Store. Replay of events is performed according to provided settings, and
+    /// read from the provided reader.
+    #[cfg(feature = "rr")]
```

Comment:
Can we add a note here (because otherwise it's only a small easy-to-miss detail in the `pub(crate)` below) that this is not the public API, and is wrapped by the `ReplayEnvironment`/`ReplayInstance` abstraction?